packages/install-expo-modules/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "../../tsconfig.node",
+  "extends": "../expo-module-scripts/tsconfig.node",
```

Comment:
nit: There's a couple of modules that import, for example `../expo-module-scripts` rather than `expo-module-scripts`. I haven't checked whether that's arbitrary or based on, for example, not adding `expo-module-scripts` to `devDependencies`. In this case, it is in `devDependencies` so seemed accidental to me.
Not crucial to change, but probably worth checking for consistency

Reply:
> Besides config collocation in one place, we can also now avoid using relative paths in config (EDIT: not in all cases, since we use ncc compiler for some packages, and it cannot correctly resolve extend config, when referring by package name, instead of path).

Again, it's a `ncc` issue looks like.